ledger/src/blockstore_processor.rs

```diff
@@ -945,17 +937,24 @@ fn load_frozen_forks(
                 cache_block_time_sender,
                 None,
                 timing,
-            )
-            .is_err()
-            {
+            );
+            // Insert even dead banks into the BankForks so that cleanup
+            // of account state created on Bank creation via `Bank::new_from_parent()`
+            // will occur when the bank is dropped. Otherwise, this state is not
+            // cleaned
+            all_banks.insert(bank.slot(), bank.clone());
+
+            if let Err(e) = process_result {
+                warn!("processing single slot {} error {:?}", slot, e);
+                initial_forks.insert(bank.slot(), bank.clone());
+                all_banks.insert(bank.slot(), bank.clone());
```

Comment:
maybe, this `insert()` into `all_banks` is redundant because it's already done at line 945?